operator/main.py

```diff
@@ -108,11 +108,11 @@ def get_brick_hostname(volname, node, suffix=True):
     # Service is created with name as Volume name. For example,
     # brick_hostname will be "server-spool1-minikube.spool1" and
     # server pod name will be "server-spool1-minikube"
-    hostname = "server-%s-%s" % (volname, node.replace(".", "-"))
+    hostname = "server-%s-%d-%s" % (volname, idx, node.replace(".", "-"))
     if suffix:
         # TODO: ordinal is used as zero. If more than one brick used from the
         # same node and for the same volume then consider changing this
-        return hostname + "-0." + volname
+        return "%s-%d.%s" % (hostname, idx, volname)
```

Comment:
This will go wrong in multi node setup. suffix is assigned by K8s. If brick pod is started in other node it will still have `-0` unless more than one brick with same pod name.

I think we can change this to `"%s-0.%s" % (hostname, volname)`  Note: idx is already part of hostname in line 111

Above TODO can be removed after this change.

Reply:
Done